src/Tooltip/index.js

```diff
@@ -15,26 +15,30 @@ const Tooltip = ({
   wrapperDisplay = "inline-flex",
   maxWidth = "400px",
   testId,
+  isOpen,
 }) => {
-  const [isOpen, setIsOpen] = useState(false);
+  const isControlled = isOpen === true || isOpen === false;
```

Comment:
```suggestion
  const isControlled = isOpen !== undefined;
```

Reply:
Your expression is more type safe - disregard unless you like the simple one better (checks for presence of the prop rather than value)